Automation/Mobile/qama.md

```diff
@@ -21,40 +21,39 @@ Check the [Appium setup](https://infinum.com/handbook/qa/automation/mobile/appiu
 
 In short, that means that:
 
-- locators are placed in the appropriate page class. 
-- tests are placed in the appropriate test class.
+- locators are placed in the appropriate page class.
+- tests are placed in the appropriate test class
```

Comment:
I see that full stops in bullet points are not present anymore in some places. Can you please reverse the deletion? In general, we are using full stops after:
- every bullet point that is a sentence.
- every bullet point that completes the introductory stem. (like this example)

I see that these rules apply to all the occurrences that are marked in the file. 😄

Reply:
I never use full stops in bullet points. Mostly because I never have full sentences and mostly do not capitalize the first letter.
And it is cleaner without punctuation.

https://www.imperial.ac.uk/brand-style-guide/writing/grammar/punctuation/bullet-points/
https://style.ons.gov.uk/house-style/bullet-points/#:~:text=you%20do%20not%20use%20full,the%20end%20of%20bullet%20points

So... just to clarify:
Do we capitalize the first letter in every bullet point?
Do we use full stop after every bullet point?
Do we use ";" after each bullet point but "." after the last?

Should there be a difference between full sentences in bullet points vs. short guidelines like in naming convention article?

Reply:
Personally, I used it in many different ways before and I wasn't consistent. Out of habit (and because I thought it looks cleaner), I mostly used the bullet structure that we commonly use when writing in Croatian (quite similar to the approach that you have), but this usage is partially incorrect for (US) English. 😅 
When taking over the QA handbook management, one of the things Neven referred to is to pay attention to the writing styles because I was doing it wrong myself. 🙈 
Unfortunately, there are quite a lot of "old" handbook articles where this isn't updated so we have a mix of everything, but we tend to keep an eye on it in newly added articles and existing articles being worked on. 

> Do we capitalize the first letter in every bullet point? --> In general, yes.
> Do we use full stop after every bullet point? --> No
> Do we use ";" after each bullet point but "." after the last?  --> Unfortunately, no 😅
> Should there be a difference between full sentences in bullet points vs. short guidelines like in naming convention article? --> If I understood correctly, I would say - yes

Here is a list of rules that sums up nicely what style we tend to have:
- Use a period or other full stop after every bullet that is a sentence. (Just like this bullet list you’re reading now.)
- Use a period after bullet list that completes the opening stem sentence that introduces it.
- Don’t use a period after bullet lists that are not complete sentences or do not complete the opening stem sentence. 
- Don’t use semicolons to end punctuation. 
- Use either all full sentences in your bullet lists or all fragments. Avoid a mix.

I copy-pasted them from [here](https://www.instructionalsolutions.com/blog/bulleted-list-punctuation) so that I don't write them by hand, but there are also similar examples [here](https://www.businesswritingblog.com/business_writing/2012/01/punctuating-bullet-points.html).

Sorry for the wall of text. Detailed grammar rules can be a pain in the ... 😄 
 

**TLDR:** Reverting the changes (and capitalizing these 2 example bullet points) will make it "correct" again.